README.md

```diff
@@ -16,46 +16,69 @@ Great for:
-    * Embedded network policy controller
+K3s is a [fully conformant](https://github.com/cncf/k8s-conformance/pulls?q=is%3Apr+k3s) Kubernetes distribution with the following changes:
+
+1. It is packaged as a single binary.
+1. It swaps out etcd for a lightweight sqlite3 storage backend. etcd3, MySQL, and Postgres are also supported.
+1. It wraps Kubernetes and other components in a single, simple launcher.
+1. It is secure by default with reasonable defaults for lightweight environments.
+1. It has minimal to no OS dependencies (just a sane kernel and cgroup mounts needed).
+
+K3s bundles the following technologies together into a single cohesive distibution:
+
+* [Containerd](https://containerd.io/) & [runc](https://github.com/opencontainers/runc)
+* [Flannel](https://github.com/coreos/flannel) for CNI
+* [CoreDNS](https://coredns.io/)
+* [Metrics Server](https://github.com/kubernetes-sigs/metrics-server)
+* [Traefik](https://containo.us/traefik/) for ingress
+* [Klipper-lb](https://github.com/rancher/klipper-lb) as an embedded service loadbalancer provider
+* [Kube-router](https://www.kube-router.io/) for network policy
+* [Helm-controller](https://github.com/rancher/helm-controller) to allow for CRD-driven deployment of helm manifests
+* [Kine](https://github.com/rancher/kine) as a datastore shim that allows etcd to be replaced with other databases
+* [Local-path-provisioner](https://github.com/rancher/local-path-provisioner) for provisioning volumes using local storage
+* [Host utilities](https://github.com/rancher/k3s-root) such as iptables/nftables, ebtables, ethtool, & socat
+
+Additionally, K3s simplifies Kubernetes operations by maintaining functionality for:
+
+* Managing the TLS certificates of Kubernetes componenents
+* Managing the connection between worker and server nodes
+* Managing an embedded etcd cluster
+* Auto-deploying Kubernetes resources from local manifests, in realtime as they are changed.
+
 
 What's with the name?
 --------------------
 We wanted an installation of Kubernetes that was half the size in terms of memory footprint. Kubernetes is a
 10 letter word stylized as k8s. So something half as big as Kubernetes would be a 5 letter word stylized as
-k3s. There is no long form of k3s and no official pronunciation.
+K3s. There is no long form of K3s and no official pronunciation.
+
+Is this a fork?
+---------------
+No. A fork implies continued divergence from the original. This is not K3s's goal or practice. K3s seeks to remain as close to upstream Kubernetes as possible. We do maintain a small set of patches (well under 1000 lines) important to K3s's usecase and deployment model. We maintain patches for other components as well. When possible, we contribute these changes back to the upstream projects, for example with [SELinux support in containerd](https://github.com/containerd/cri/pull/1487/commits/24209b91bf361e131478d15cfea1ab05694dc3eb). This is a common practice amongst software distributions.
```

Comment:
I'd call out that it is not the intention of k3s to change kubernetes to add or change behavior of its core features.  The changes are primarily done to assist how we package. I know you already said half of that.